application-templates/starter/package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "build": "mc-scripts build",
     "start": "dotenv -- mc-scripts start",
-    "start:prod:local": "NODE_ENV=production MC_APP_ENV=development dotenv -- mc-http-server --use-local-assets",
+    "compile-html": "NODE_ENV=production dotenv -- mc-scripts compile-html",
+    "compile-html:local": "NODE_ENV=production MC_APP_ENV=development dotenv -- mc-scripts compile-html --transformer @commercetools-frontend/mc-dev-authentication/transformer-local.js",
```

Comment:
This is turning out to be pretty nice. The `mc-dev-authentication` package now implements a `transformer-local.js` that compiles and copies the login/logout routes to the `public` folder of the application.

This way, everything is still static and we can start the app in production mode locally.

Reply:
What is also nice is that we can reference npm packages/modules.

```
--transformer @commercetools-frontend/mc-dev-authentication/transformer-local.js
```

Reply:
This is also a bit related to #1733.
We can have a package with pre-configured transformers, which people can also simply copy or extend.